test/testutil/compare.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017-2025 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "../testutil.h"
+
+static void strip_line_ends(char *str)
+{
+    size_t i;
+
+    for (i = strlen(str);
+         i > 0 && (str[i - 1] == '\n' || str[i - 1] == '\r');
+         i--);
+
+    str[i] = '\0';
+}
+
+int compare_ssl_trace_with_file(BIO *membio, const char *reffile)
```

Comment:
Maybe rename to `compare_with_reference_file` or some such as this looks like a quite generic template file comparison now that it is factored out into a separate file.

Reply:
Okay I can make that change.